irepela/homework_6/unpack.py

```diff
@@ -0,0 +1,38 @@
+def unpack_while_loop(items: list) -> list:
+    """
+      Flattens list using while
+
+      Args:
+          items: nested list of items
+
+      Returns:
+          list: flattened list
+    """
+    unpacked = []
+    copied_list = list(reversed(items))
+    while len(copied_list) > 0:
+        next_item = copied_list.pop()
+        if isinstance(next_item, list):
+            copied_list.extend(reversed(next_item))
+        else:
+            unpacked.append(next_item)
+    return unpacked
```

Comment:
nice, 
so many reversed, you can try to remove them and make final reversed at the end